python/dgl/heterograph_index.py

```diff
@@ -58,10 +61,26 @@ def get_relation_graph(self, etype):
         Returns
         -------
         HeteroGraphIndex
-            The bipartite graph.
+            The unitgraph graph.
         """
         return _CAPI_DGLHeteroGetRelationGraph(self, int(etype))
 
+    def flatten_relations(self, etypes):
+        """Convert the list of requested unitgraph graphs into a single unitgraph
```

Comment:
Unit graph is an _internal_ data structure representing the basic computing block of heterograph. It is either a normal graph or a uni-directional bipartite graph, so it can be represented by one sparse matrix (not necessarily square).